### DISCUSSION TITLE
Objects aren't appearing on the Collection site

### DISCUSSION THREAD

**Discussion by TheFernMagician** [CollectionBuilder/collectionbuilder.github.io#64] [Q&A]
I've been going through the tutorial and I've only been able to get the changes I made to the URL, base URL, title, tagline, and description variables in the config.yml to work/display on my personal collection site. I have uploaded 20 jpgs to the objects folder, I have uploaded a csv to the data folder with the required elements filled out: objectid, filename, title, format -- I also have my descriptions in, and I have adjusted the required fields under "Collection Settings" in the config.yml. I even re-uploaded my objects after compressing them to save on space, but I just can't get my site to display anything from my objects or my csv (under "browse" I have 0 of 0 objects). And GH is not displaying any errors after running everything through Jekyll. I've gone back through everything to check for spelling/formatting and I don't see any errors. Any ideas on where to look next? Thanks for any and all help!

**Comment by evanwill** [comment]
@TheFernMagician I just took a look at your repository--> your metadata and objects work (and look delicious!!). The issue is that somehow your "_config.yml" got corrupted. I honestly don't know how or why, and can't seem to fix it by just editing the file. I deleted your existing version, copied in a fresh version from CB-GH, and it just works fine. This fix is in a [pull request](https://github.com/TheFernMagician/Home-Cooking/pull/1)

**Reply by evanwill** [reply]
@TheFernMagician could you tell us what editor you were using to edit the "_config.yml"? I have seen a similar issue once before, and I am trying to figure it out--really mysterious!

**Reply by evanwill** [reply]
@TheFernMagician wait! Scratch the mystery!
It is just extra space in the YAML... I saw the extra space in front of `url` and `baseurl`, but there was also an space in front of `noindex` which was breaking things. So your "_config.yml" wasn't mysteriously corrupted--it just had extra spaces in front of a few of the keys, which breaks Jekyll (and Jekyll wasn't giving helpful error messages...).

**Reply by TheFernMagician** [reply]
Thank you so much for the help! Can I delete the old/corrupted config file? As far as the editor, I'm not quite sure what you mean -- I just hit the pencil button on the original config file and edited in-browser (using Firefox with a few plugins: Dark Reader, DuckDuckGo Privacy Essentials, HTTPS Everywhere, Privacy Badger, and uBlock Origin). I'm not sure if this answers your question, let me know!

**Reply by TheFernMagician** [reply]
Wow those spaces make the difference, huh! Crazy! Thank you!

**Reply by evanwill** [reply]
Yeah, in YAML tiny details can break everything (that's Jekyll + Ruby, not CollectionBuilder issue)... sorry for my confusion, it wasn't actually too complicated issue--you were doing everything right, except for three little spaces!

Yes, you can delete the old config!

**Reply by TheFernMagician** [reply]
Thank you so much for the help today, I really appreciate it!
